src/java.base/share/classes/java/util/concurrent/atomic/AtomicReferenceFieldUpdater.java

```diff
@@ -46,7 +46,7 @@
 
 /**
  * A reflection-based utility that enables atomic updates to
- * designated {@code volatile} reference fields of designated
+ * designated non-static {@code volatile} reference fields of designated
```

Comment:
Wouldn't this change also apply to the other AtomicXFieldUpdaters?

Reply:
Yes. Will do. Also, as mentioned by @liach we could trap violations by throwing a better exception. But considering that people should be using VarHandles instead anyway these days, and old uses might depend on current behavior, it doesn't seem worthwhile to add? (line 338)
+            if (Modifier.isStatic(modifiers))
+                throw new IllegalArgumentException("Must not be static");